lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -453,29 +459,83 @@ abstract class PrimaryRoleManager {
   @visibleForTesting
   List<Role> get debugSecondaryRoles => _secondaryRoleManagers?.map((RoleManager manager) => manager.role).toList() ?? const <Role>[];
 
+  @protected
+  DomElement createElement() => domDocument.createElement('flt-semantics');
+
+  DomElement _createAndInitElement() {
```

Comment:
nit: actually, I'd still keep this `static` and pass all the information it needs to initialize the element (I think all it needs is the element created by `createElement` and `semanticsObject`). This way this method will not need access to `this`.